File: java/claude/P19.java
// Task p19: Reads value from command line and adds 1000 to it
public class P19 {
    public static void main(String[] args) {
        if (args.length > 0) {
            try {
                int value = Integer.parseInt(args[0]);
                int result = value + 1000;
                System.out.println("Input: " + value);
                System.out.println("Result (value + 1000): " + result);
            } catch (NumberFormatException e) {
                System.out.println("Invalid input. Please provide a valid integer.");
            }
        } else {
            System.out.println("Usage: java P19 <number>");
        }
    }
}

File: javascript/claude/p19.js
// Task p19: Reads value from command line and adds 1000 to it
if (process.argv.length > 2) {
  try {
    const value = parseInt(process.argv[2]);
    const result = value + 1000;
    console.log(`Input: ${value}`);
    console.log(`Result (value + 1000): ${result}`);
  } catch (e) {
    console.log("Invalid input. Please provide a valid integer.");
  }
} else {
  console.log("Usage: node p19.js <number>");
}
